### PR TITLE
fix(chatbar-mobile): fixed chatbar focus when pressed

### PR DESCRIPTION
**Pull request by ThomBos**
**What this PR does** 📖
Fixes the issue that occurs when a mobile user presses on the chatbar (it loses focus when trying to type a message and closes the keyboard)
**Which issue(s) this PR fixes** 🔨
AP-1910

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Comment by WanderingHogan** [comment]
https://user-images.githubusercontent.com/2993032/175180930-bd0d5519-e1b6-465b-8024-bfd3bffe21cc.mp4

**Comment by ThomBos** [comment]
> telegram-cloud-document-1-5017426283185832685.mp4

reverse typing is a feature, not a bug :joy:

i'm testing again, but should be all solved

File: components/ui/Chat/Scroll/Scroll.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div
   :class="classObject"
   ref="scrollRef"
-  v-on:scroll="onScrolled(); hideKeyboard();"
+  v-on:scroll="onScrolled()"
```
Comment (molimauro) [inline components/ui/Chat/Scroll/Scroll.html:4]: `@scroll`

**Review by molimauro — CHANGES_REQUESTED** [review]

File: components/views/chat/message/Message.vue [hunk 2]
```diff
@@ -255,7 +255,9 @@ export default Vue.extend({
         messageID: this.$props.message.id,
         to: to === myTextilePublicKey ? from : to,
       })
-      this.$store.dispatch('ui/setChatbarFocus')
+      setTimeout(() => {
+        this.$store.dispatch('ui/setChatbarFocus')
+      }, 150)
```
[the remark this answers is not in the document]
Comment (ThomBos) [reply components/views/chat/message/Message.vue:260]: this fixes a reply issue that caused the chatbar to lose focus when the reply button was pressed (due to v-click-outside)

[the remark this answers is not in the document]
Comment (ThomBos) [reply components/views/chat/message/Message.vue:260]: the chatbar appears and you have to close it and press back again, to make it work

**Review by molimauro — APPROVED** [review]

File: components/views/chat/message/Message.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (ThomBos) [reply components/views/chat/message/Message.vue:260]: > Could we go for nextTick rather than timeout?
idk, i go check if it works 👍

**Review by phillsatellite — APPROVED** [review]

**Review by stavares843 — APPROVED** [review]

**Pull request merged**